hudi-spark-datasource/hudi-spark3.3.x/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/Spark33LegacyHoodieParquetFileFormat.scala

```diff
@@ -271,7 +255,7 @@ class Spark33LegacyHoodieParquetFileFormat(private val shouldAppendPartitionValu
               DataSourceUtils.int96RebaseSpec(footerFileMetaData.getKeyValueMetaData.get, int96RebaseModeInRead)
             val datetimeRebaseSpec =
               DataSourceUtils.datetimeRebaseSpec(footerFileMetaData.getKeyValueMetaData.get, datetimeRebaseModeInRead)
-            new Spark32PlusHoodieVectorizedParquetRecordReader(
+            new Spark3HoodieVectorizedParquetRecordReader(
```

Comment:
Could you file a ticket to look at Spark's parquet file format class in recent Spark versions and see if there is any change that need to ported to our file format class?

Reply:
We extend sparks vectorized reader so it will be up to date

Reply:
Sg